global-news-app/src/Utils/consts.ts

```diff
@@ -1,3 +1,35 @@
 export const SelectWidth = 175;
 export const dateFormat = "EEEE MMM dd, yyyy";
 export const datePickerFormat = "dd-MM-yy";
+export const COLORS = [
```

Comment:
i would move this to the file where the colors and spacing are. you can change the name to CHARTS_COLORS or any other that is appropriate